src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Cohort/ConfirmProviderRequest.cs

```diff
@@ -1,8 +1,24 @@
-﻿namespace SFA.DAS.EmployerCommitmentsV2.Web.Models.Cohort
+﻿using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.EmployerCommitmentsV2.Web.Extensions;
+
+namespace SFA.DAS.EmployerCommitmentsV2.Web.Models.Cohort
 {
-    public class ConfirmProviderRequest : SelectProviderRequest
+    public class ConfirmProviderRequest : BaseConfirmProviderRequest
+    {
+        public long AccountLegalEntityId { get; set; }
+    }
+
+    public class BaseConfirmProviderRequest : IndexRequest
     {
         public long ProviderId { get; set; }
```

Comment:
Again, I think ProviderId should be hidden from the URL as well.

Reply:
provider id is entered by the user in the journey